src/views/bouquets/BouquetEditView.vue

```diff
@@ -93,20 +97,47 @@ const onSubmit = async () => {
     ...form.value,
     datasets: datasets.value.map(d => d.dataset.id),
   }
+
+  const checkWhiteSpacing = (val) => {
+    if (val.trim() === '') {
+      noWhiteSpace.value = `Merci de bien remplir les champs`
+      return true
+    }
+    return false
+  };
+
   if (isCreate) {
+    if (checkWhiteSpacing(data.name) || checkWhiteSpacing(data.description)) {
+      return;
+    }
+
     res = await bouquetStore.create({
       ...data,
       tags: [config.universe_name],
       extras: extrasFromDatasets(),
     })
   } else {
+    if (checkWhiteSpacing(data.name) || checkWhiteSpacing(data.description)) {
+      return;
+    }
+
     res = await bouquetStore.update(loadedBouquet.value.id, {
       ...data,
       tags: loadedBouquet.value.tags,
       extras: { ...loadedBouquet.value.extras, ...extrasFromDatasets()},
     })
   }
-  router.push({ name: "bouquet_detail", params: { bid: res.slug } })
+  
+  isFormValidated.value = true;
```

Comment:
Why not use the error states from each field? It could make the logic cleaner and be more useful to the user.